README.md

```diff
@@ -291,114 +291,112 @@ Library specific options:
+#### c) Using your own compiled libraries
 
 **This is outdated and not recommended**
 
 Follow the instructions on https://dev.minetest.net/Build_Win32_Minetest_including_all_required_libraries#VS2012_Build
 
-### Windows Installer using WIX Toolset
+### Windows Installer using WiX Toolset
 
 Requirements:
-* Visual Studio 2017
-* Wix Toolset
+* [Visual Studio 2017](https://visualstudio.microsoft.com/)
+* [WiX Toolset](https://wixtoolset.org/)
 
-In Visual Studio 2017 Installer select "Optional Features" -> "Wix Toolset"
+In the Visual Studio 2017 Installer, select **Optional Features -> WiX Toolset**.
 
-Build the binaries like described above, but make sure you unselect "RUN_IN_PLACE".
+Build the binaries like described above, but make sure you unselect `RUN_IN_PLACE`.
 
-Open the generated Project file with VS. Right click "PACKAGE" and choose "Generate".
+Open the generated project file with Visual Studio. Right-click **Package** and choose **Generate**.
 It may take some minutes to generate the installer.
 
 
 Docker
 ------
-We provide Minetest server docker images using the Gitlab mirror registry.
+We provide Minetest server Docker images using the GitLab mirror registry.
 
 Images are built on each commit and available using the following tag scheme:
 
 * `registry.gitlab.com/minetest/minetest/server:latest` (latest build)
 * `registry.gitlab.com/minetest/minetest/server:<branch/tag>` (current branch or current tag)
 * `registry.gitlab.com/minetest/minetest/server:<commit-id>` (current commit id)
 
-If you want to test it on a docker server, you can easily run:
+If you want to test it on a Docker server, you can easily run:
 
 	sudo docker run registry.gitlab.com/minetest/minetest/server:<docker tag>
 
-If you want to use it in a production environment you should use volumes bound to the docker host
+If you want to use it in a production environment, you should use volumes bound to the Docker host
 to persist data and modify the configuration:
 
 	sudo docker create -v /home/minetest/data/:/var/lib/minetest/ -v /home/minetest/conf/:/etc/minetest/ registry.gitlab.com/minetest/minetest/server:master
 
 Data will be written to `/home/minetest/data` on the host, and configuration will be read from `/home/minetest/conf/minetest.conf`.
 
-Note: If you don't understand the previous commands, please read the official Docker documentation before use.
+**Note:** If you don't understand the previous commands, please read the official Docker documentation before use.
 
-You can also host your minetest server inside a Kubernetes cluster. See our example implementation in `misc/kubernetes.yml`.
+You can also host your Minetest server inside a Kubernetes cluster. See our example implementation in [`misc/kubernetes.yml`](misc/kubernetes.yml).
 
 
 Version scheme
 --------------
-We use `major.minor.patch` since 5.0.0-dev. Prior to that we used `0.major.minor`.
+We use `major.minor.patch` since 5.0.0-dev. Prior to that, we used `0.major.minor`.
```

Comment:
Added comma not needed.

Reply:
I think it sounds much more natural to have a comma here, as "we used…" introduces a different proposition.

Reply:
I can see why it's tempting, but it's worse grammar. I'm a native English speaker and writer.
Anyway, @Ezhh will know better than either of us =)